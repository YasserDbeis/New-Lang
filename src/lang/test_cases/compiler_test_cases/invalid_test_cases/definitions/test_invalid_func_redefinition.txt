func main() {

}

func foo() {

}

func foo() {
    
}