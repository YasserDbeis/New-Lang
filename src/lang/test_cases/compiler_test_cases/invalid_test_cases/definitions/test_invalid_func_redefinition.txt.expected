RUNTIME EXCEPTION
Error occurred in Execution
Table entry 'foo' already defined
Function already defined
-1